examples/stream_l1_marketdata.py

```diff
@@ -24,17 +24,19 @@ async def main():
             "POPCAT-USDT BINANCE Perpetual/USDT Crypto*BINANCE-FUTURES-USD-M/DIRECT"
         ],
     ):
-        market_name = "<unknown>"
```

Comment:
line 21 in this,
```python
    async for snap in c.subscribe_l1_book_snapshots(
```
has the type error
```
"Coroutine[Any, Any, AsyncIterator[L1BookSnapshot]]" is not iterable
  "__aiter__" method not defined
```



`subscribe_l1_book_snapshots` (supposedly) returns a `Coroutine[Any, Any, AsyncIterator[L1BookSnapshot]]`
This does not pass the type check because it is using `async for `directly on a coroutine without awaiting it first.


If this truly returned `Coroutine[Any, Any, AsyncIterator[L1BookSnapshot]]`, then it should be more like
```python
await snaps = c.subscribe_l1_book_snapshots(
        "binance-futures-usd-m.marketdata.architect.co",
        market_ids=[
            "POPCAT-USDT BINANCE Perpetual/USDT Crypto*BINANCE-FUTURES-USD-M/DIRECT"
        ],
)
async for snap in snaps:
    [...]
```